.github/workflows/scb-bot.yaml

```diff
@@ -78,10 +79,10 @@ jobs:
 
       - name: Fetch new release changelog
         if: ${{ env.release != env.local && env.release != null }}
-        # Fetches changelog and filters out any issue references
+        # Reformats the versionApi to have an HTML view of the release changelog
         run: |
-          changelog=$(curl -sL ${{env.versionApi}} | jq -r ".body")
-          echo 'releaseChangelog=$changelog' >> $GITHUB_ENV
+          changelog=$(echo ${{env.versionApi}} | sed -e 's/api.//g;s/\/repos//g;s/latest//g')
```

Comment:
Can you add a comment (maybe with an example) on what this replace looks like?
Had some problems properly understanding what this is doing

Reply:
Done